### PR TITLE
Kill and disable xscreensaver along with other unnecessary autostart programs

### PR DESCRIPTION
**Pull request by duxovni**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -29,6 +29,11 @@ cat $HERE/nginx-site | sed "s|DIR|$MZ_LOCATION|" | sudo tee /etc/nginx/sites-ava
 sudo ln -sf /etc/nginx/sites-available/musicazoo /etc/nginx/sites-enabled/musicazoo
 
 
+echo "=> Disabling unwanted programs"
+killall -q xscreensaver
+echo -n > $HOME/.config/lxsession/LXDE/autostart
```
Comment (celskeggs) [inline install.sh]: This is a bit broad -- I'd rather not just arbitrarily delete the entire contents of a configuration file that isn't xscreensaver-specific.

Comment (sdukhovni) [reply install.sh]: My rationale is that we really don't need much of a desktop environment at all as long as VLC can enter full screen, so none of LXDE's autostart programs are needed.

Comment (celskeggs) [reply install.sh]: It's possible that someone might install this on a system that they also use for something else, so I still don't like blowing away an entire configuration file.

**Comment by duxovni** [comment]
Finally picking this back up again. The install script now only removes xscreensaver from LDXE's autostart.

**Pull request merged**